### PR TITLE
Run the adopting steps recursively

### PR DESCRIPTION
**Pull request by nox**
When adopting a <template>'s template contents and running its own adopting steps as defined in HTML, if the adopting steps aren't recursively, any nested template won't see its template contents' owner document changed.

### CODE REVIEW

**Comment by nox** [comment]
No idea what that AttrExodus comment is about.

File: dom.bs [hunk 1]
```diff
@@ -4637,14 +4637,16 @@ a <var>document</var>, run these steps:
...
- <li>Run any <a>adopting steps</a> defined for
- <var>node</var> in <a>other applicable specifications</a> and pass
- <var>node</var> and <var>oldDocument</var> as parameters.
+ <li>
+  For each <var>descendant</var> in <var>nodes</var>'s <a>inclusive descendants</a>, in
+  <a>tree order</a>, run these substeps:
+
+  <ol>
+   <li>Set <var>descendant</var>'s <a>node document</a> to <var>document</var>.
+   <!--AttrExodus as well as any associated {{Attr}} nodes-->
```
Comment (annevk) [inline dom.bs]: This needs to be followed by a newline.

**Comment by annevk** [comment]
AttrExodus is for when we find out we need to revive the concept of `Attr` as a `Node`.

**Comment by nox** [comment]
Added the new line.

**Pull request closed without merging**